modules/oxide/src/less/theme/components/form/layout.less

```diff
@@ -69,6 +69,7 @@
   .tox-form__controls-h-stack {
     align-items: center;
     display: flex;
+    height: 60px;
```

Comment:
The implications of a fixed height has to be throughly researched.

Reply:
This comes from the existing image edit panel styles; and yes it will expand other dialogs but anywhere a `sizeinput` is used inside a `bar` the height was _already_ this big. What winds up happening without this is our upcoming partial redial to show `sizeinput` in the existing `bar` causes the dialog to jiggle.

The aim of this PR is to get feedback and come up with solutions. I'm not sure what else to do.
https://github.com/tinymce/tinymce/blob/c51299435dd3be85ac41db4cfe326043faf978d3/modules/oxide/src/less/theme/alien/unknowns.less#L58-L64

Reply:
It's still causing issues when used elsewhere as I mentioned though, so we need something else here. I'd suggest we make this tied specifically to imagetools by adding a imagetools class to the panel we're using.

Reply:
There needs to be a setting to add that class to the `bar` panel though. I started with adding a `toolbar` property to `bar` components but then I couldn't figure out how to require (or automatically change) any `button` components to be toolbar buttons.

Maybe call it `expanded` or something similar instead?

Reply:
Why does it need to be on the `bar`? You should be able to do something like this:
```
.tox-imagetools .tox-bar.tox-form__controls-h-stack {
  height: 60px;
}
```

Reply:
The `h-stack` _is_ the bar. But how the change is done in CSS is an implementation detail - I'm talking about the API. Where should I add to the spec something that says the `h-stack` should be 60px high?

Reply:
You're misunderstanding me... you **don't** need anything in the API on the bar to do this. This can be done solely via CSS as it always has been.

Reply:
Also don't forget `.tox-form__controls-h-stack` is used by other things, not just for the `bar` component.

Reply:
Discussed off-thread. This was all a huge misunderstanding because I didn't know `panel` components supported custom classes in the API spec. So what Lee has been asking for is to add a custom class via the API and then use that to increase the height when a `bar` component is inside a panel with that custom class. That technique might also negate the need for a specific spacer component; I'll give that a go as well.